### ISSUE TITLE
Display name for each row in a bed file

### ISSUE THREAD

**Issue by Imipenem** [deeptools/pyGenomeTracks#323]
- [x] Search whether this issue (or a similar issue) has been solved before
  using the search tab above. Link the previous issue if appropriate below.

  - [x] Paste your pyGenomeTracks version (`pyGenomeTracks --version`) and your python
  version (`python --version`) below.
   `pyGenomeTracks version = 3.6, python version = 3.9.1`

  - [x] Paste the full pyGenomeTracks command that produces the issue below
  (ignore if you simply spotted the issue in the code/documentation).
`pyGenomeTracks --tracks bed_tracks.ini --region 6:90636247-91006627 --width 38 --dpi 130 -o myimage.png
`

Hey, I'm trying to use your package with a bed file that looks like the following:
`#chrom  start   end     name    score   strand
chr6    90637876        91003900        myname   0       -
... and so on
`
Everything worked fine and the image is generated.
However, I noticed that the "names" were not displayed for each row as specified in my bed file. Is this somehow possible to do? I saw it in the docs but was not able to reproduce it.
So basically I'd like to place `myname` next to its graphical representation in the image (if any), like it's shown in the docs.

Many thanks ;)

**Comment by LeilyR** [comment]
How does your your bed file block in the ini file look?

**Comment by Imipenem** [comment]
`file = myfile.bed 
height = 7 
title = my title
fontsize = 10 
style = UCSC`

It's basically just copied from the docs

**Comment by lldelisle** [comment]
If your region goes to the right of the plotting area, you may be interested by the options:
```
all_labels_inside = true
labels_in_margin = true
```
See: https://github.com/deeptools/pyGenomeTracks/blob/master/pygenometracks/tests/test_data/master_bed_all_label_inside.png

**Comment by LeilyR** [comment]
names should be added by default and your blocks seems fine too. You may try to add the options @lldelisle proposed and see if that would do the job

**Comment by Imipenem** [comment]
This worked, thanks a lot.
